### PR TITLE
Add gpt configs

### PR DESCRIPTION
**Pull request by maciek-pioro**
# Description
todo

# Checklist
   - [] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [] Are all variables set with logical defaults that are backward compatible?
   - [] I attached link to neptune if it was necessary
   - [] Have you successfully executed the linter and tests?
   - [] Are all functions concise and don't require splitting?
   - [] Is there documentation for arguments and complex logic?
   - [] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [] Are all functions placed in appropriate files?
   - [] Is the PR name meaningful and descriptive?
   - [] Is PR description provided, or unnecessary?
  
Other questions:
   - [] If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters
   - [] If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)
   - [] If there is a need for a second reviewer I requested additional review

### CODE REVIEW

File: research/conditional/train/configs/baselines/common.yaml
Comment (Simontwice) [inline research/conditional/train/configs/baselines/common.yaml:17]: I say `25_000`

Comment (Simontwice) [inline research/conditional/train/configs/baselines/common.yaml:17]: I say `25_000`

File: research/conditional/train/configs/baselines/common.yaml
Comment (Simontwice) [inline research/conditional/train/configs/baselines/common.yaml:25]: I think the best way to implement scheduling would be to say the final learning rate and when it should happen. Like: 1/100 the original lr, in 200K steps

Comment (maciek-pioro) [reply research/conditional/train/configs/baselines/common.yaml:25]: Yup, a different PR will fix that. It will be merged before this one. https://trello.com/c/Egpuus34/261-fix-scheduling-remember-to-add-cosine-and-constant-at-the-end

Comment (Simontwice) [reply research/conditional/train/configs/baselines/common.yaml:25]: roger that.

File: research/conditional/train/configs/baselines/common.yaml
Comment (ludziej) [reply research/conditional/train/configs/baselines/common.yaml]: change save_weights_path to "weights.ckpt", it was my typo some time ago. Also it does not save weights at the end of the training (in practice it probably does, if number of iterations divides save_weights_interval).

File: research/conditional/train/configs/baselines/common.yaml
Comment (sebastianjaszczur) [inline research/conditional/train/configs/baselines/common.yaml:13]: yeah yeah, f them

Comment (sebastianjaszczur) [reply research/conditional/train/configs/baselines/common.yaml:13]: BTW, should we change names in child configs as well? Basically in each and every config?

**Review by sebastianjaszczur — COMMENTED** [review]
LGTM, but I wait for other people's approval

File: research/conditional/train/configs/baselines/common.yaml
Comment (Simontwice) [reply research/conditional/train/configs/baselines/common.yaml:13]: yes, the name here does not matter too much.

File: research/conditional/train/configs/baselines/common.yaml
Comment (Simontwice) [reply research/conditional/train/configs/baselines/common.yaml:23]: 5%

File: research/conditional/train/configs/baselines/common.yaml
Comment (ludziej) [reply research/conditional/train/configs/baselines/common.yaml]: also, add neptune run id to yaml name

**Review by Simontwice — APPROVED** [review]

File: research/conditional/train/configs/baselines/common.yaml
Comment (Simontwice) [reply research/conditional/train/configs/baselines/common.yaml]: No .ckpt suffixes allowed anymore. It handles step number already.

**Pull request merged**
